src/redux/features/Plantings/plantingsThunks.ts

```diff
@@ -0,0 +1,26 @@
+import { Dispatch } from 'redux';
+import { PlantingsService } from 'src/services';
+import { RootState } from 'src/redux/rootReducer';
+import { setPlantingsAction } from './plantingsSlice';
+
+export const requestPlantings = (organizationId: number, plantingSiteId: number) => {
+  return async (dispatch: Dispatch, _getState: () => RootState) => {
+    try {
+      const response = await PlantingsService.listPlantings(organizationId, [
+        {
```

Comment:
thoughts on just passing the org id and planting site id into the listPlantings api? have the service populate the search criteria.. unless we will be using the search criteria and sort order in some upcoming work

Reply:
I'll look into this, thanks